app/views/shared/_document.html.erb

```diff
@@ -2,9 +2,23 @@
   <div class="aspect-ratio-4x3">
     <div class="aspect-ratio-content">
       <img src="<%= document[:thumbnail_src] %>">
+      <% document.response[:highlighting][document.id][:text].tap do |highlights| %>
+        <% highlights.reject do |highlight|
+             term = highlight.match(/<em>([^<]*)<\/em>/)[1].downcase
+             document[:title].downcase.include?(term)
+             # If someone is doing a known item search,
+             # repeating the title just looks odd.
+        end.each do |highlight| %>
+          <div class="highlight">
+            &hellip;
+            <%= highlight.html_safe %>
+            &hellip;
+          </div>
+        <% end if highlights %>
+      <% end if document.class.method_defined? :response %>
```

Comment:
@mccalluc I think `document.respond_to? :response` would be a more common idiom here.

But yeah, if `document` doesn't respond to `:response`, then I think that's a sign of something more problematic, don't you? Do you know what `document` is in those cases?